### ISSUE TITLE
Make it limit responses per thread

### ISSUE THREAD

**Issue by Shivang44** [shieric4/coy-bot#1]
It's a pretty good bot but for example on one J&A thread it replied to 3 different replies. It should probably limit to 1-2 responses per thread, otherwise it looks spammy.

**Comment by fictionic** [comment]
I agree. Except if someone replies to the bot, it should be allowed to continue the quote.

I suppose ideally it would only reply to replies if the follow-up quote was a follow-up to the original quote... but that sounds like a lot more work. If I want it that badly I'll implement it lol
